api/internal/platform/api/api.go

```diff
@@ -81,3 +82,13 @@ func BadRequest(code int, text string) Response {
 		Status: http.StatusBadRequest,
 	}
 }
+
+func NotFound(code int, text string) Response {
```

Comment:
```suggestion
func NotFound(text string) Response {
```

One good practice is "design interfaces for default usage". In this case, most likely we will never want to specify any other code except `CodeNotFound`. So we can just omit the argument and always set `CodeNotFound` in the response.